presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/rule/PushPartialAggregationThroughExchange.java

```diff
@@ -255,4 +264,13 @@ private PlanNode split(AggregationNode node, Context context)
                 node.getHashSymbol(),
                 node.getGroupIdSymbol());
     }
+
+    private boolean isLambda(RowExpression rowExpression)
```

Comment:
Can this be a static method?

Another similar methods is `MultipleDistinctAggregationToMarkDistinct#toSymbol`. Should they be put in a Util class? @highker